python/main.py

```diff
@@ -41,5 +74,52 @@ async def get_image(image_name):
     if not image.exists():
         logger.debug(f"Image not found: {image}")
         image = images / "default.jpg"
-
     return FileResponse(image)
+
+# JSONファイルから商品データを読み込む関数
+def load_items() -> List[Dict]:
+    with open("items.json", "r") as file:
+        data = json.load(file)
+    return data["items"]
```

Comment:
fyi, you can use `read_items_from_json` only.
Usually redundant functions are not recommended.